test/groovy/templates/PiperInitRunStageConfigurationTest.groovy

```diff
@@ -192,16 +175,6 @@ steps: {}
             stageConfigResource: 'testDefault.yml'
         )
 
-        assertThat(nullScript.commonPipelineEnvironment.configuration.runStage.keySet(),
```

Comment:
Why have these assertions been removed?

Reply:
Ah I thought the below ones already cover it but they only assert for the `runStep` values. 
Actually I wanted to replace those assertions with assertions on the actual values for each stage, because here it is only ensured that the runStage map contains values for the stages but they might be set to false instead of true.